client/payment-details/summary/index.tsx

```diff
@@ -50,6 +50,8 @@ import WCPaySettingsContext from '../../settings/wcpay-settings-context';
 import { FraudOutcome } from '../../types/fraud-outcome';
 import CancelAuthorizationButton from '../../components/cancel-authorization-button';
 import { PaymentIntent } from '../../types/payment-intents';
+import MissingOrderNotice from 'wcpay/payment-details/summary/missing-order-notice';
+import _ from 'lodash';
```

Comment:
Lodash is an external dependency, can you move it to the top of the file please?